### DISCUSSION TITLE
can't run Webassembly

### DISCUSSION THREAD

**Discussion by ramsay77** [unoplatform/uno#17067] [Q&A]
.nuget\packages\uno.wasm.bootstrap\8.0.15\build\Uno.Wasm.Bootstrap.targets(213,3): error : [00:00:00.3797646] Failed to clone the Emscripten SDK repository. Review the error logs for additional information and check https://aka.platform.uno/msbuild-troubleshoot
11:29:30:898	1>

\.nuget\packages\uno.wasm.bootstrap\8.0.15\build\Uno.Wasm.Bootstrap.targets(213,3): error : System.Exception: Failed to run emscripten. More details are available in diagnostics logging mode or using the MSBuild /bl switch, see https://aka.platform.uno/msbuild-troubleshoot.
11:29:30:898	1>
\.nuget\packages\uno.wasm.bootstrap\8.0.15\build\Uno.Wasm.Bootstrap.targets(213,3): error :    at Uno.Wasm.Bootstrap.ShellTask_v1f66b9b15b66efa87ca84d70b6879ddd12d18414.RunPackager() in /_/src/Uno.Wasm.Bootstrap/ShellTask.cs:line 768
11:29:30:898	1>

nuget\packages\uno.wasm.bootstrap\8.0.15\build\Uno.Wasm.Bootstrap.targets(213,3): error :    at Uno.Wasm.Bootstrap.ShellTask_v1f66b9b15b66efa87ca84d70b6879ddd12d18414.Execute() in /_/src/Uno.Wasm.Bootstrap/ShellTask.cs:line 271

Hello guys i want to run webassembly uno projekt i have created a new uno platform project, installed uno templates, and even installed emsdk manuel but when i want to run webassembly it want to clone the repo even if i installed it and set the %PATH% variable any idee guys?

**Comment by jeromelaban** [comment]
Thanks for the report. You can troubleshoot this error using the link provided above. It's likely that git is not configured properly, though, the error message will tell more.

**Comment by smalgin** [comment]
I switched MSBuild to Diagnostic mode, still not helpful:

1>        C:\Users\serge\.nuget\packages\uno.wasm.bootstrap\8.0.15\build\Uno.Wasm.Bootstrap.targets(213,3): error : System.Exception: Failed to run emscripten. More details are available in diagnostics logging mode or using the MSBuild /bl switch, see https://aka.platform.uno/msbuild-troubleshoot.
1>        C:\Users\serge\.nuget\packages\uno.wasm.bootstrap\8.0.15\build\Uno.Wasm.Bootstrap.targets(213,3): error :    at Uno.Wasm.Bootstrap.ShellTask_v1f66b9b15b66efa87ca84d70b6879ddd12d18414.RunPackager() in /_/src/Uno.Wasm.Bootstrap/ShellTask.cs:line 768
1>        C:\Users\serge\.nuget\packages\uno.wasm.bootstrap\8.0.15\build\Uno.Wasm.Bootstrap.targets(213,3): error :    at Uno.Wasm.Bootstrap.ShellTask_v1f66b9b15b66efa87ca84d70b6879ddd12d18414.Execute() in /_/src/Uno.Wasm.Bootstrap/ShellTask.cs:line 271
1>      Done executing task "ShellTask_v1f66b9b15b66efa87ca84d70b6879ddd12d18414" -- FAILED.
1>    Done building target "BuildDist" in project "Commerce.csproj" -- FAILED.



I also deleted <userDir>/.uno, that didn't help

**Reply by jeromelaban** [reply]
Thanks for the update. Can you provide a [binlog of the build](https://platform.uno/docs/articles/uno-builds-troubleshooting.html)?

**Reply by smalgin** [reply]
@jeromelaban - Thanks for asking. As I was simply running samples, I simply switched my PC.. let me try the old box again and see if that still a problem there.

**Reply by smalgin** [reply]
That involves updating VS, running uno-check, updating VS extensions and nuget packages on the samples... I'll keep you posted.

**Reply by smalgin** [reply]
@jeromelaban I updated VS 2022 Community to latest, Uno Platform VS extension to latest, uno-check to latest. Cannot reproduce the emscripten issue anymore. If I find it again, I will make sure to save binlog files. Thank you!
